content/02.main-text.md

```diff
@@ -158,7 +158,7 @@ We would encourage the maintainers of similar projects to consider broader codes
 Open writing presents new opportunities for scholarly communication.
 `TODO: reference "paper of the future"? arXiv:1601.02927 doi:10.22541/au.149693987.70506124 doi:10.22541/au.148769949.92783646 http://blogs.nature.com/naturejobs/2017/06/01/techblog-c-titus-brown-predicting-the-paper-of-the-future`
 Though it is still valuable to have versioned drafts of a review manuscript with digital identifiers, journal publication may not be the terminal endpoint for collaborative manuscripts.
-After releasing the first version of our collaborative review [@doi:10.1101/142760], six new authors have contributed text (Figure @fig:contrib) and existing authors continue to discuss new literature, [creating a living document](https://github.com/greenelab/deep-review/).
+After releasing the first version of our collaborative review [@doi:10.1101/142760], {{deep_review_post_submission_authors}} new authors have contributed text (Figure @fig:contrib) and existing authors continue to discuss new literature, [creating a living document](https://github.com/greenelab/deep-review/).
```

Comment:
One more thought: maybe we should say `{{deep_review_post_submission_authors}} new contributors have updated the manuscript` because some of the new contributors may not be listed as authors.

Reply:
Hi @agitter - yes confirming the github username change from avctrh. Sorry about any automation breakage!